core/zstd/src/ZipZSTD.cxx

```diff
@@ -35,9 +35,11 @@ void R__zipZSTD(int cxlevel, int *srcsize, char *src, int *tgtsize, char *tgt, i
                                         2*cxlevel);
 
     if (R__unlikely(ZSTD_isError(retval))) {
-        std::cerr << "Error in zip ZSTD. Type = " << ZSTD_getErrorName(retval) <<
-        " . Code = " << retval << std::endl;
-        return;
+        if (R__unlikely(retval != static_cast<size_t>(errorCodeSmallBuffer))) {
```

Comment:
```suggestion
        if (R__unlikely(retval != errorCodeSmallBuffer)) {
```

Reply:
Actually:
```
if (R__unlikely(ZSTD_isError(retval))) {
  if (R__unlikely(retval != static_cast<size_t>(errorCodeSmallBuffer))) {
```
should be combined as
```
if (R__unlikely(ZSTD_isError(retval) && retval != static_cast<size_t>(errorCodeSmallBuffer))) {
```
(i.e. have `R__unlikely` surround the whole condition)